### PR TITLE
Team33 37 signup backend

### PR DESCRIPTION
**Pull request by ValeSer**
Progress setting up database: working order

### CODE REVIEW

File: backend/src/mongoconfig.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  // doc.name = "Testin insert";
+
+  // doc.save().then(() => {
+  //   console.log('saved');
+  // }).catch((error) => {
+  //   console.log(error);
+  // });
+});
+
+export default db;
```
Comment (robimaliqi) [inline backend/src/mongoconfig.ts:24]: This looks good

**Review by robimaliqi — COMMENTED** [review]
Finished review

File: backend/src/mongoconfig.ts (hunk 1 above)
Comment (ValeSer) [reply backend/src/mongoconfig.ts:24]: thank you

**Review by bjentwistle — APPROVED** [review]
Looks good

**Pull request merged**